toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -94,9 +128,15 @@ if [[ -f "$DOCKERCONTAINERONLY" ]]; then
 fi
 
 echo "Done installing all packages, creating $chroot_archive." | tee -a "$chroot_log"
+
+record_timestamp "done installing all packages"
+
 if command -v pigz &>/dev/null ; then
     tar -I pigz -cvf "$chroot_archive" -C "$chroot_base/$chroot_name" . >> "$chroot_log"
 else
     tar -I gzip -cvf "$chroot_archive" -C "$chroot_base/$chroot_name" . >> "$chroot_log"
 fi
 echo "Done creating $chroot_archive." | tee -a "$chroot_log"
+
+record_timestamp "done packing installations"
```

Comment:
Calling this the chroot is more common for the Mariner teams, so it's going to be easier to understand.

```suggestion
record_timestamp "done installing all packages"

record_timestamp "start packing the chroot"

if command -v pigz &>/dev/null ; then
    tar -I pigz -cvf "$chroot_archive" -C "$chroot_base/$chroot_name" . >> "$chroot_log"
else
    tar -I gzip -cvf "$chroot_archive" -C "$chroot_base/$chroot_name" . >> "$chroot_log"
fi
echo "Done creating $chroot_archive." | tee -a "$chroot_log"

record_timestamp "done packing the chroot"
```